code/nx.json

```diff
@@ -3,8 +3,10 @@
   "extends": "nx/presets/npm.json",
   "nxCloudAccessToken": "NGVmYTkxMmItYzY3OS00MjkxLTk1ZDktZDFmYTFmNmVlNGY4fHJlYWQ=",
   "defaultBase": "next",
-  "useLegacyCache": true,
```

Comment:
@kasperpeulen Do you remember the reason of why we have introduced the legacy cache initially?